### PR TITLE
Fixed bug #20 - doesn't crash when phone is set to Chinese

### PR DESCRIPTION
**Pull request by kungpaogao**
### Summary

This pull request fixes the bug in issue #20. The bug caused the app to crash when the phone's language is set to Chinese, and the user navigates to the facility page.

### Test Plan

Tested on my Huawei Mate 8. Might need additional testing with other devices.

### CODE REVIEW

File: Density/app/src/main/java/org/cornelldti/density/density/util/ColorBarMarkerView.kt [hunk 1]
```diff
@@ -15,6 +15,7 @@ import com.github.mikephil.charting.data.Entry
 import com.github.mikephil.charting.highlight.Highlight
 
 import org.cornelldti.density.density.R
+import java.util.*
```
Comment (SamChou19815) [inline Density/app/src/main/java/org/cornelldti/density/density/util/ColorBarMarkerView.kt]: This change looks extra

Comment (kungpaogao) [reply Density/app/src/main/java/org/cornelldti/density/density/util/ColorBarMarkerView.kt]: True, I'm not sure why it was added. I will try fixing and building again.

Comment (kungpaogao) [reply Density/app/src/main/java/org/cornelldti/density/density/util/ColorBarMarkerView.kt]: Ok, it's used for `Date`, `Calendar`, and `Locale` so I've just imported them on their own. It's reflected in my latest PR.

**Comment by kungpaogao** [comment]
Issues are resolved in a more recent PR

**Pull request closed without merging**